visualization/app/codeCharta/util/treeMapHelper.ts

```diff
@@ -28,6 +28,39 @@ export class TreeMapHelper {
 		return geomMap
 	}
 
+	public static buildRootFolderForFixedFolders(map: CodeMapNode, heightScale: number, state: State, isDeltaState: boolean): Node {
+		const flattened: boolean = this.isNodeToBeFlat(map, state)
+		const height = 2
+
+		return {
+			name: map.name,
+			id: 0,
+			width: 100,
+			height: 2,
+			length: 100,
```

Comment:
While we use `width` and `height` in the `.cc.json`, we convert the term `height` to `length` since height is defined as the height of a building in a 3-dimensional space.
That's why the height and length are 100 here, which are the boundaries defined in the cc.json

Reply:
Could we rename the building height to e.g., `buildingHeigth`? That seems more intuitive than naming it `length`?

Reply:
![image](https://user-images.githubusercontent.com/12533626/92534508-47cc9600-f235-11ea-89a6-a8b8c58a465d.png)

building height is already called `height`. I hope this illustration is clear

Reply:
Ah! That does indeed clarify the naming! Thanks!